dockerfile-captain.release

```diff
@@ -1,21 +1,17 @@
 FROM node:18-alpine
-RUN apk update && apk upgrade --no-cache && apk add --update --no-cache make gcc g++ git curl openssl openssh
+RUN apk add --update --no-cache make gcc g++ git curl openssl openssh
 
-RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
-
+COPY . ./
 
 # Build backend code
-
-COPY . /usr/src/app
-
 RUN npm ci && \
```

Comment:
shouldn't this be changed to `npm ci --omit=dev` ?

Reply:
`npm ci --omit=dev` is present in line no. 10

Reply:
The --omit=dev flag in npm ci command does not remove dev dependencies that were previously installed. Instead, it instructs npm ci not to install dev dependencies during its execution.

You probably need to run `rm -rf node_modules` before running this to actually achieve the size reduction.

Reply:
No. `npm ci --omit=dev` first removes `node_modules` folder if present and then only install packages for production.

Reply:
TIL! Thanks!

Reply:
Correct. But we don't release only for a small set of additional features, once a few other major changes are added we'll publish the next major release version. Currently there are only minor updates. https://github.com/caprover/caprover/blob/master/CHANGELOG.md#next-version---available-as-edge

Reply:
Great